x11-libs/xapps/xapps-2.0.5.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8,9} )
+VALA_USE_DEPEND="vapigen"
+
+inherit gnome2-utils vala meson python-r1 xdg-utils
+
+DESCRIPTION="Cross-desktop libraries and common resources"
+HOMEPAGE="https://github.com/linuxmint/xapp/"
+LICENSE="GPL-3"
+
+SRC_URI="https://github.com/linuxmint/xapp/archive/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/xapp-${PV}"
+KEYWORDS="~amd64 ~arm64 ~x86"
+
+SLOT="0"
+IUSE="gtk-doc introspection static-libs"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	>=dev-libs/glib-2.44.0:2
+	dev-libs/gobject-introspection:0=
+	dev-libs/libdbusmenu[gtk3]
+	gnome-base/libgnomekbd
+	x11-libs/cairo
+	>=x11-libs/gdk-pixbuf-2.22.0:2[introspection?]
+	>=x11-libs/gtk+-3.16.0:3[introspection?]
+	x11-libs/libxkbfile
+"
+DEPEND="${RDEPEND}
+	$(vala_depend)
+	sys-devel/gettext
+
+	gtk-doc? ( dev-util/gtk-doc )
+"
+BDEPEND="
+	${PYTHON_DEPS}
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	dev-util/gdbus-codegen
+	dev-util/glib-utils
+	>=dev-util/intltool-0.40.6
+	sys-devel/gettext
+"
+
+PATCHES=(
+	# Use proper GTK modules env variable
+	# https://github.com/linuxmint/xapp/commit/e846e76a392d44d211b7007691358f173f1b44a3
+	"${FILESDIR}/${PN}-2.0.5-fix-gtk-module-env.patch"
+)
+
+src_prepare() {
+	xdg_environment_reset
+	vala_src_prepare
+	default
+}
+
+src_configure() {
+	local emesonargs=(
+		$(meson_use gtk-doc docs)
+		-Dpy-overrides-dir="/pygobject"
+	)
+	meson_src_configure
+}
+
+src_install() {
+	meson_src_install
+	rm -rf "${ED}/usr/bin" || die
+
+	# package provides .pc files
+	find "${D}" -name '*.la' -delete || die
+
+	# copy pygobject files to each active python target
+	# work-around for "py-overrides-dir" only supporting a single target
+	install_pygobject_override() {
+		PYTHON_GI_OVERRIDESDIR=$("${PYTHON}" -c 'import gi;print(gi._overridesdir)') || die
+		einfo "gobject overrides directory: $PYTHON_GI_OVERRIDESDIR"
+		mkdir -p "${ED}/$PYTHON_GI_OVERRIDESDIR/"
+		cp -r "${D}"/pygobject/* "${ED}/$PYTHON_GI_OVERRIDESDIR/" || die
+		python_optimize "${ED}/$PYTHON_GI_OVERRIDESDIR/"
+	}
+	python_foreach_impl install_pygobject_override
+	rm -rf "${D}/pygobject" || die
```

Comment:
As explained before, drop the `-f`. Although maybe for this one it's fine, but try without. We want to see if something doesn't go as planned.